src/game/CRamp.cpp

```diff
@@ -62,9 +62,9 @@ CAbstractActor *CRamp::EndScript() {
         maskBits |= kSolidBit + kDoorIgnoreBit;
 
         deltaY = ReadFixedVar(iDeltaY);
-        dims[0] = FMulDivNZ(gLastBoxRect.right - gLastBoxRect.left, FIX(5), 72);
+        dims[0] = FIX(gLastBoxRect.right - gLastBoxRect.left);
```

Comment:
for example this code... if we are in Fixed units we don't need the FIX(...) on the difference.